test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua

```diff
@@ -0,0 +1,59 @@
+--  Requirement summary:
+--  [RegisterAppInterface] REJECT
+--  [RegisterAppInterface] RegisterAppInterface and HMILevel
+--
+--  Description:
+--  Check that App will be rejected with HMI type WEB_VIEW 
+--  when policy table is empty.
```

Comment:
@ydementieiev This description of test script is incorrect. SDL can not start with empty policy table.
I think you mean that mobile application has no policy record in local policy table here. 
Please correct test script description.

Reply:
@aderiabin thanks you ! [done ](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/9271d45b0b4e176fad4cd4070eeb060facb11f43#diff-ff5652b3ded34af081249da10ca6203fR7)